### ISSUE TITLE
Imagenet

### ISSUE THREAD

**Issue by PushparajaMurugan** [VDIGPKU/DADA#6]
Hi. Unable to reproduce the results (search_gumbel). I got only 65.23% (top1) in imagenet.?

**Comment by latstars** [comment]
Would you like to provide us the searching and training logs to help find out the problems?

**Comment by PushparajaMurugan** [comment]
Hi @latstars, 

pls have a look at the log files.

[Search_log.txt](https://github.com/VDIGPKU/DADA/files/5194714/Search_log.txt)
[imaenet_training_log-20200901-015111.txt](https://github.com/VDIGPKU/DADA/files/5194715/imaenet_training_log-20200901-015111.txt)

thanks,
Raja.

**Comment by latstars** [comment]
> Hi @latstars,
> 
> pls have a look at the log files.
> 
> [Search_log.txt](https://github.com/VDIGPKU/DADA/files/5194714/Search_log.txt)
> [imaenet_training_log-20200901-015111.txt](https://github.com/VDIGPKU/DADA/files/5194715/imaenet_training_log-20200901-015111.txt)
> 
> thanks,
> Raja.

Hi, we find the learning rate schedule in your imagenet-training log is not the same with our schedule. Maybe you should check your pytorch version and the training code.

**Comment by PushparajaMurugan** [comment]
Hi @latstars . Thanks, Let me train the network again.
